Rakefile

```diff
@@ -36,7 +46,7 @@ task :clean do
   sh "xcodebuild clean -configuration Release -scheme #{SCHEME} -destination '#{DESTINATION}' | xcpretty -c; exit ${PIPESTATUS[0]}"
 end
 
-task :default => ["test:specs", "test:demo"]
+task :default => ["test:package", "test:specs", "test:demo"]
```

Comment:
With the introduction of testing building with the Swift PM here, I think it would be neat to have a separate rake task, perhaps just `rake test` so that developers can run _both_ the unit tests and 'Demo' tests together, but separate from testing building through the Swift PM.

Do you think that would be useful? Would you be cool to add that in?